library/build.gradle

```diff
@@ -15,6 +15,17 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
+        checkTypes {
+            javaCompileOptions.annotationProcessorOptions.
+                    classNames.add("org.checkerframework.checker.keystore.KeyStoreChecker")
```

Comment:
For the upstream of the keystore checker (my repo), here we need to add another one configuration. But if you want to test these benchmarks with your repo, it is fine for me.